packages/rancher-istio/1.25/rancher-istio/charts/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Istio
+  catalog.cattle.io/kube-version: '>= 1.28.0-0 < 1.33.0-0'
```

Comment:
For the rancher 2.11 line this should be >=1.30.0

Reply:
good catch